WordPress/Classes/Plugins/Views/PluginListItemView.swift

```diff
@@ -3,68 +3,105 @@ import SwiftUI
 import AsyncImageKit
 import WordPressAPI
 import WordPressCore
+import SafariServices
 
 struct PluginListItemView: View {
 
     @ScaledMetric(relativeTo: .body) var descriptionFontSize: CGFloat = 14
+    @State private var showingSafariView = false
```

Comment:
Have you tried `.subheadline` instead of `descriptionFontSize`? It should be close enough.